Sources/Orbit/Components/Timeline.swift

```diff
@@ -77,31 +80,31 @@ struct TimelinePreviews: PreviewProvider {
             TimelineStep(
                 "Booked",
                 sublabel: "January 3, 10:43",
-                style: .status(.success),
+                style: .pastSuccess,
```

Comment:
For the snapshot, one of the steps should be "active" for completeness (we probably have some more exhaustive in the previews already that can be used for snapshot)

Reply:
I added the active state and face some weird layout in the snapshots (you can check in the diffs). Do you know what might be causing it or how it can be fixed? In the previews all looks fine
<img width="432" alt="image" src="https://user-images.githubusercontent.com/13184609/199034209-98a7ec72-ec82-4147-94a9-1434a68ce7c5.png">

And another question: can we more to Orbit `IsPreviewLayoutScreenContentKey`? Or is there something similar already? I'd like to have more control over animation state if it's for a snapshot

Reply:
I will check the layout. For the `IsPreviewLayoutScreenContentKey`, it might be ported in future, but would it alone help? Here, we just need to decide what animation state we want to have as "starting" one (non-animated version) and loop the animation around that point. (snapshots test code can simply force reduceMotion environment to false in order to have the state non-animated)

Reply:
For the snapshots, this is probably animation issue - try moving the frame modifier after the animation modifier.

Reply:
The state should be done so that both devices with reduced motion and snapshots show the same thing.

It seems to look fine for `Timeline` component, but not for `TimelineItem` alone - not an issue though, as long as the `Timeline` itself works fine.

Reply:
The animation is fixed by adding what value to observe for animation